StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/STPAPIClient+Link.swift

```diff
@@ -434,6 +435,9 @@ extension STPPaymentMethodBillingDetails {
 
     var consumersAPIParams: [String: Any] {
         var params = STPFormEncoder.dictionary(forObject: self)
+        // Consumers API doesn't support email or phone.
+        params["email"] = nil
+        params["phone"] = nil
```

Comment:
I think this only removes the value, not the key, right? Seems like we should remove both key and value?

Reply:
setting to nil removes the value, if there is no value there is no key